flow.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
```

Comment:
If these definitions are only used within `fusion-core`, what are your thoughts on not adding it to the the `[lib]` config and instead importing the specific types in the files that need it?  Benefit of not polluting the global namespace and avoiding any collision issues down the road.  Downside is more imports...

Reply:
I was having trouble with that approach combined with method overrides